tests/test_importvalue.py

```diff
@@ -2,13 +2,16 @@
 from pygeppetto.constants import UserPrivileges
```

Comment:
@filippomc shall we add a test for importtypes?

Reply:
Good point, I'll do it

Reply:
@tarelli test added, it was really useful to spot a problem that has been then solved.